### PR TITLE
fix: register worker

### PR DESCRIPTION
**Pull request by UncleBill**

### CODE REVIEW

File: packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx [hunk 1]
```diff
@@ -141,10 +141,6 @@ export function AccountView() {
                     <TableBody>
                         {userTokenBalances?.map((balance: UserIdeaTokenBalance) => {
                             const token = balance.token
-                            const name =
-                                balance.token.name.length > 30
-                                    ? balance.token.name.slice(0, 30).concat('...')
-                                    : balance.token.name
```
Comment (SebastianLF) [inline packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx:147]: Why this part is removed ?

Comment (UncleBill) [reply packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx:147]: name is never used.

Comment (UncleBill) [reply packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx:147]: Besides, you should use https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow to truncate the text.

Comment (SebastianLF) [reply packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx:147]: Got it.

**Pull request merged**